### PR TITLE
160 sign in fixes: navigate to Dashboard when clicking magic link and added sign out button

### PR DESCRIPTION
**Pull request by acao22**
### 160 sign in fixes: navigate to Dashboard when clicking magic link and added sign out button

**Description:** This PR closes out most of the items on issue #160. Here are my changes:
1. Directly navigate to Dashboard when a user clicks on the magic link instead of back to the sign in screen. There are a few instances where a notification pops up saying that the magic link is expired or incorrect when the website actually logs you in successfully, but I think that might be a different error (maybe related to issue #125 ) or just something different between production and dev mode
2. Added a signout button in `Dashboard.tsx` under settings
3. Added button styling (border, hover, and some drop shadow) for the signout and delete account buttons in settings so that users know it is clickable

### CODE REVIEW

File: client/public/locales/en/translation.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "answers": "Answers",
     "browse-list": "Browse the list of statements you have answered so far",
     "delete-account": "Delete account",
+    "signout": "Sign out",
```
Comment (amirrr) [inline client/public/locales/en/translation.json:71]: Can you also add other translations for the ones we have to this?

File: client/src/components/Enter.tsx [hunk 2]
```diff
@@ -4,7 +4,11 @@ import { useParams, useNavigate } from "react-router-dom";
 import Navbar from "../partials/NavBar";
 import Footer from "../partials/Footer";
 
-const Enter = ({ signIn }) => {
+interface EnterProps {
```
Comment (amirrr) [inline client/src/components/Enter.tsx:7]: Use `type` in most places where you don't need to extend

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

**Review by amirrr — DISMISSED** [review]
Well done!

File: client/public/locales/en/translation.json (hunk 1 above)
Comment (acao22) [reply client/public/locales/en/translation.json:71]: yep, adding right now

**Review by amirrr — APPROVED** [review]

**Pull request merged**